src/components/FolderPicker/helpers.js

```diff
@@ -0,0 +1,47 @@
+import { getParentPath } from 'lib/path'
+
+const isInvalidMoveTarget = (subjects, target) => {
+  const isASubject = subjects.find(subject => subject._id === target._id)
+  const isAFile = target.type === 'file'
+  return isAFile || isASubject !== undefined
+}
+
+const getParentFolderFromPath = path => {
+  if (path === '/') return undefined
+
+  const newPath = path.split('/').slice(0, -1)
+  if (newPath.length === 1) {
+    return {
+      name: 'root',
+      path: '/'
+    }
+  }
+
+  return {
+    name: newPath[newPath.length - 1],
+    path: newPath.join('/')
+  }
+}
```

Comment:
```suggestion
/**
 * Gets the parent folder information from a given path
 * @param {string} path - The folder path
 * @returns {Object|undefined} - The parent folder information
 */
const getParentFolderFromPath = path => {
  if (path === '/') return undefined;

  const newPath = path.split('/').slice(0, -1);
  const parentPath = newPath.join('/');

  return {
    name: newPath.length === 1 ? 'root' : newPath[newPath.length - 1],
    path: parentPath || '/'
  };
};
```

Reply:
After converting to typescript I no longer needed this function, but it's a good suggestion for improvement